src/js/i18n/locales/en.json

```diff
@@ -34,5 +34,6 @@
   "Home.Secondarycontent.bodytext": "Alpha.austin.gov is a new website and a work in progress. For the full City of Austin website, visit  {citySiteLink}. Learn more about the new website at {projectsSiteLink}.",
   "Service.RelatedLinks.SectionHeader": "Check out related services",
   "Service.RelatedLinks.Tag": "Service",
-  "Home.RelatedLinks.Tag": "Service"
+  "Home.RelatedLinks.Tag": "Service",
+  "Submenu.workInProgress": "Alpha.austin.gov is a work in progress. For the full City of Austin website, visit {citySiteLink}."
```

Comment:
We'll have to add Submenu.workInProgress for all language json files. This we can grab this from the first half of Home.Secondarycontent.bodytext? It might get weird in vi, I dunno.